.github/workflows/security_codeql.yml

```diff
@@ -0,0 +1,38 @@
+name: CodeQL
+on:
+  workflow_call:
+
+jobs:
+  codeql:
+    name: CodeQL Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript', 'python' ]
+        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python', 'ruby' ]
+        # Learn more about CodeQL language support at https://aka.ms/codeql-docs/language-support
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      # Initializes the CodeQL tools for scanning.
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v2
+        with:
+          languages: ${{ matrix.language }}
```

Comment:
Instead of making this action run twice (once for JavaScript and once for Python), you can run it once with both languages.

`languages: python, javascript`

See the code snippet right above [this anchor](https://github.com/github/codeql-action#configuration-file).

Reply:
Yes you can, and I thought about that, but actually liked the idea of having it separate; since we effectively break our code/language by area (python:backend, js:frontend). The time required is the same with minimum overhead on each; so might save 10 seconds on the build/cleanup. I'll play with a few other cleanup steps to and see where it goes; There are a few other break-outs I'd prefer to do across the pipeline as well.

Reply:
Ah word. Sounds good.